File: src/main/java/baboon/industry/block/storage/entity/TileEntityBatboxHV.java
package baboon.industry.block.storage.entity;

import baboon.industry.IndustryConfig;

public class TileEntityBatboxHV extends TileEntityBatboxBase {

    public TileEntityBatboxHV() {
        setCapacity(IndustryConfig.cfg.getInt("Energy Values.hvBatteryStorage") * 3);
        setTransfer(IndustryConfig.cfg.getInt("Energy Values.hvIO"));
        setMaxReceive(IndustryConfig.cfg.getInt("Energy Values.hvIO"));
    }

    @Override
    public String getInvName() {
        return "IndustryBatboxHV";
    }
}

File: src/main/kotlin/turniplabs/industry/blocks/entities/batbox/TileEntityBatboxHV.kt
package turniplabs.industry.blocks.entities.batbox

import sunsetsatellite.sunsetutils.util.Connection
import sunsetsatellite.sunsetutils.util.Direction
import turniplabs.industry.Industry2
import turniplabs.industry.IndustryConfig

class TileEntityBatboxHV : TileEntityBatboxBase() {

    init {
        this.setCapacity(65536)
        this.setTransfer(IndustryConfig.cfg.getInt("Energy Values.highVoltage"))
        this.setMaxReceive(IndustryConfig.cfg.getInt("Energy Values.highVoltage"))
    }

    override fun getInvName(): String {
        return "BatboxHV"
    }
}